netebpfext/net_ebpf_ext_tracelog.h

```diff
@@ -122,3 +110,25 @@ net_ebpf_ext_trace_terminate();
         TraceLoggingKeyword((keyword)),                                       \
         TraceLoggingString(message, "Message"),                               \
         TraceLoggingUInt64((value), (#value)));
+
+#define NET_EBPF_EXT_RETURN_RESULT(status)                 \
```

Comment:
These look (I think) to be the same as old lines 69-89... why the move?

Reply:
These seemed out of place, sandwiched between the NET_EBPF_EXT_LOG_xxx macros.  So I thought the logging macros would seem more 'readable' if they were all contiguous and that these were located after the logging ones.  It's all very subjective, of course :-)